docker-compose.yml

```diff
@@ -25,6 +25,7 @@ x-sentry-defaults: &sentry_defaults
   environment:
     SENTRY_CONF: '/etc/sentry'
     SNUBA: 'http://snuba-api:1218'
+    SENTRY_EVENT_RETENTION_DAYS: 
```

Comment:
Don't you uhh, need a value here?

Reply:
Nope. This is how you pass the value defined on the host to the container/service. The default value of this var is defined in `.env` and we want to pass that value.

Reply:
Is it confusing because of the trailing colon?

Reply:
It definitely is but I don't really know a better way :(

Reply:
I forget ... can we put comments in YAML? It's JSON where we can't really comment, right?

Worth a comment here?

(I mean, this is already merged, but ... )

Reply:
FOO: null

Would be equivalent and look at least intentional.

Reply:
Added the comments in #759 which is the follow up.

I don't want `FOO: null` as it also looks confusing: did you want to unset this? What? Why?